cabal-install/tests/IntegrationTests/regression/common.sh

```diff
@@ -0,0 +1,9 @@
+# Helper to run Cabal
+cabal() {
+    "$CABAL" $CABAL_ARGS "$@"
```

Comment:
These files are no longer needed unless you need to override the default common.sh (which it does not seem to be the case.)